_maps/map_files/Mining/Lavaland.dmm

```diff
@@ -76,9 +76,6 @@
 "an" = (
 /turf/closed/mineral/random/labormineral/volcanic,
 /area/lavaland/surface/outdoors)
-"ao" = (
```

Comment:
Keys getting deleted?

Reply:
`ao` is a duplicate key of `ai`, they're both `(/turf/closed/mineral/random/volcanic, /area/lavaland/surface/outdoors)`.

Reply:
Yes, but why is it getting deleted? Did you not mapmerge?

Reply:
Currently trialling some improvements to mapmerge, I need to PR and get some eyes on it (like figuring out if this behavior is a good idea or not), in the meantime I can undo these changes if desired.

Reply:
It seems pointless if you're not going to write to those unused keys afterward and continue end of the sequence. 

Gaps in the dictionary without the ability to reuse the old keys causes unintended behavior with the current version of MM since it doesn't see the dictionary as full when you get to the end of the sequence and will overflow/corrupt the map when the resorted dictionary surpasses ZZ.